File: sdk/python-tunzisha/add_patient_data.py
import requests
import json

url = "https://3000-kcelestinom-tunzishaxch-gy1ysd1zmzc.ws-eu108.gitpod.io/api/Patients"

payload = json.dumps({
  "patientId": 125,
  "firstName": "Dennis",
  "secondName": "Otieno",
  "age": 25,
  "gender": "Male",
  "profileImage": "YAYGSLSMNJAMAN",
  "contactNumber": "+254743992235",
  "nextOfKin": "Michael Omolo",
  "dateOfBirth": "01/02/1999",
  "inpatient": True,
  "symptoms": "breathlessness coughing dry_throat",
  "diagnosis": "Covid-19 Omicron",
  "prescription": "Vitamin C and Zinc tablets, each 1 x 3"
})
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

File: sdk/java-tunzisha/add_patient_data.java
public class add_patient_data {

    public static void main(String[] args) {

        OkHttpClient client = new OkHttpClient().newBuilder()
                .build();
        MediaType mediaType = MediaType.parse("application/json");
        RequestBody body = RequestBody.create(mediaType,
                "{\r\n    \"patientId\": 125,\r\n    \"firstName\": \"Dennis\",\r\n    \"secondName\": \"Otieno\",\r\n    \"age\": 25,\r\n    \"gender\": \"Male\",\r\n    \"profileImage\": \"YAYGSLSMNJAMAN\",\r\n    \"contactNumber\": \"+254743992235\",\r\n    \"nextOfKin\": \"Michael Omolo\",\r\n    \"dateOfBirth\": \"01/02/1999\",\r\n    \"inpatient\": true,\r\n    \"symptoms\": \"breathlessness coughing dry_throat\",\r\n    \"diagnosis\": \"Covid-19 Omicron\",\r\n    \"prescription\": \"Vitamin C and Zinc tablets, each 1 x 3\"\r\n}");
        Request request = new Request.Builder()
                .url("https://3000-kcelestinom-tunzishaxch-gy1ysd1zmzc.ws-eu108.gitpod.io/api/Patients")
                .method("POST", body)
                .addHeader("Content-Type", "application/json")
                .build();
        Response response = client.newCall(request).execute();
    }
}